### DISCUSSION TITLE
Update to Chinese (Traditional) locale [zh_TW]

### DISCUSSION THREAD

**Discussion by SeasonChannel32** [aclap-dev/video-downloadhelper#822] [Language & Translation]
Translation for Video DownloadHelper Version 9.0.2.3
have edit to be clearer some sentences
[zh_TW.json](https://github.com/user-attachments/files/15586069/zh_TW.json)
[zh_TW.txt](https://github.com/user-attachments/files/15586076/zh_TW.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language.

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
